packages/ag-charts-website/e2e/toolbar.spec.ts

```diff
@@ -139,14 +139,17 @@ test.describe('toolbar', () => {
 
     test('AG-13008 delete annotation', async ({ page }) => {
         await gotoExample(page, url);
+        const { wrapper } = await locateCanvas(page);
         const point = await canvasToPageTransformer(page);
         const hover = point(200, 200);
         const leave = point(300, 400);
 
         // Test 1. Check that the Delete & Backspace keys work:
         await page.locator(SELECTORS.textAnnotationMenu).click();
         await page.locator(SELECTORS.commentMenuItem).click();
+        await page.mouse.move(hover.x, hover.y);
```

Comment:
Is this `move` necessary? There are no other moves in the steps after this which is unusual.

Reply:
Maybe not, but it reflects user behaviour better, since clicks don't just appear in the middle of the canvas without some hover events beforehand.